chipsec/helper/linux/linuxhelper.py

```diff
@@ -77,10 +77,10 @@
 IOCTL_MSGBUS_SEND_MESSAGE      = 0x15
 IOCTL_FREE_PHYSMEM             = 0x16
 
-LZMA  = os.path.join(chipsec.file.TOOLS_DIR,"compression","bin","LzmaCompress")
-TIANO = os.path.join(chipsec.file.TOOLS_DIR,"compression","bin","TianoCompress")
-EFI   = os.path.join(chipsec.file.TOOLS_DIR,"compression","bin","TianoCompress")
-BROTLI = os.path.join(chipsec.file.TOOLS_DIR,"compression","bin","Brotli")
+LZMA  = os.path.join(os.path.dirname(sys.argv[0]),chipsec.file.TOOLS_DIR,"compression","bin","LzmaCompress")
+TIANO = os.path.join(os.path.dirname(sys.argv[0]),chipsec.file.TOOLS_DIR,"compression","bin","TianoCompress")
+EFI   = os.path.join(os.path.dirname(sys.argv[0]),chipsec.file.TOOLS_DIR,"compression","bin","TianoCompress")
+BROTLI = os.path.join(os.path.dirname(sys.argv[0]),chipsec.file.TOOLS_DIR,"compression","bin","Brotli")
```

Comment:
You may want to rebase this branch against the latest master.  This change has been merged already.  Should not need to be in this change.

Reply:
Sorry, didn't see tha this was on the Python 3 branch.  Ignore.